Source/Applications/Backend/applications/ApplicationQueries.ts

```diff
@@ -6,6 +6,11 @@ import { ILogger } from '@shared/backend/logging';
 import { injectable } from 'tsyringe';
 import { Application, ApplicationModel } from './Application';
 
+import { NamespaceList } from '@shared/backend/k8s/NamespaceList';
```

Comment:
I get a squiggly in vscode under this import with the following message:
>Output file '/Users/pavneet/code/Dolittle/Studio/Source/Shared/Backend/dist/k8s/NamespaceList.d.ts' has not been built from source file >'/Users/pavneet/code/Dolittle/Studio/Source/Shared/Backend/k8s/NamespaceList.ts'.ts(6305)

It builds just fine, but I suspect we need to export the k8s shared modules from the backend?

Reply:
I need to do some investigations on this. I saw this yesterday locally as well and did a `yarn tsc -b` in the Shared/Backend folder and it went away. My guess is that something is not configured correctly. 
I'll dig a bit.

Reply:
oh! `yarn tsc -b` fixed it for me as well. It also fixed the `withProjectionFor` squiggles I've been getting. So is there something with the typescript projects that isn't updating the built shared projects?

Reply:
Solving this outside this PR